pkg/cmd/util.go

```diff
@@ -56,3 +58,40 @@ func getApplicationsFromFiles(files []string) ([]inputData, error) {
 	}
 	return appData, nil
 }
+
+// GetAllYMLFiles if path in argument is directory get all *.yml and *.yaml files
+// in that directory. If path is file just add it to output list as it is.
+func GetAllYMLFiles(paths []string) ([]string, error) {
+	var files []string
+
+	for _, path := range paths {
+		fileInfo, err := os.Stat(path)
+		if err != nil {
+			return nil, errors.Wrapf(err, "can't get file info about %s", path)
+		}
+		if fileInfo.IsDir() {
+			ymlFiles, err := filepath.Glob(filepath.Join(path, "*.yml"))
+			if err != nil {
+				return nil, errors.Wrapf(err, "can't list *.yml files in %s", path)
+			}
+			files = append(files, ymlFiles...)
+
+			yamlFiles, err := filepath.Glob(filepath.Join(path, "*.yaml"))
+			if err != nil {
+				return nil, errors.Wrapf(err, "can't list *.yml files in %s", path)
+			}
+			files = append(files, yamlFiles...)
+
+		} else {
+			// path is regular file, do nothing and just add it to list of files
+			files = append(files, path)
+		}
+
+	}
+
```

Comment:
For me, it makes the code more readable, but whatever, I've removed the lines. I don't want to argue about empty lines.